tests/test_coinmoves.py

```diff
@@ -1117,6 +1117,286 @@ def test_coinmoves_unilateral_htlc_dust(node_factory, bitcoind):
+                         'debit_msat': 0,
+                         'extra_tags': [],
+                         'originating_account': fundchannel['channel_id'],
+                         'output_msat': 49864547000,
+                         'primary_tag': 'to_them',
+                         'utxo': f"{only_one(close_info['txids'])}:{to_l1}"},
+                        {'account_id': 'wallet',
+                         'blockheight': 104,
+                         'credit_msat': 50000000000,
+                         'debit_msat': 0,
+                         'extra_tags': [],
+                         'originating_account': fundchannel['channel_id'],
+                         'output_msat': 50000000000,
+                         'primary_tag': 'deposit',
+                         'utxo': f"{only_one(close_info['txids'])}:{to_l2}"}]
+    check_channel_moves(l1, expected_channel1)
+    check_chain_moves(l1, expected_chain1)
+    check_channel_moves(l2, expected_channel2)
+    check_chain_moves(l2, expected_chain2)
+
+    bitcoind.generate_block(1, wait_for_mempool=1)
+    line = l2.daemon.wait_for_log('Resolved THEIR_UNILATERAL/THEIR_HTLC by our proposal THEIR_HTLC_FULFILL_TO_US')
+    htlc_success_txid = re.search(r'by our proposal THEIR_HTLC_FULFILL_TO_US \(([0-9a-f]{64})\)', line).group(1)
+
+    expected_chain1 += [{'account_id': 'external',
+                         'blockheight': 104,
+                         'credit_msat': 100000000,
+                         'debit_msat': 0,
+                         'extra_tags': [],
+                         'originating_account': fundchannel['channel_id'],
+                         'output_msat': 100000000,
+                         'payment_hash': inv['payment_hash'],
+                         'primary_tag': 'htlc_fulfill',
+                         'utxo': f"{only_one(close_info['txids'])}:{htlc}"},
+                        {'account_id': 'external',
+                         'blockheight': 105,
+                         'credit_msat': 0,
+                         'debit_msat': 0,
+                         'extra_tags': [],
+                         'originating_account': fundchannel['channel_id'],
+                         'output_msat': 100000000,
+                         'spending_txid': htlc_success_txid,
+                         'primary_tag': 'htlc_fulfill',
+                         'utxo': f"{only_one(close_info['txids'])}:{htlc}"}]
+    # FIXME: I expect channel events here, like so:
+    # expected_channel1 += [{'account_id': fundchannel['channel_id'],
+    #                        'credit_msat': 0,
+    #                        'debit_msat': 100000000,
+    #                        'fees_msat': 0,
+    #                        'group_id': 1,
+    #                        'part_id': 0,
+    #                        'payment_hash': inv['payment_hash'],
+    #                        'primary_tag': 'invoice'}]
+    # expected_channel2 += [{'account_id': fundchannel['channel_id'],
```

Comment:
this is a duplicate of the onchain event at L1324 that credits the account via an onchain movement?